src/index.ts

```diff
@@ -16,6 +16,14 @@ const fromNullable = <T>(v: T | Nullable): Maybe<T> => {
   return nothing();
 };
 
+/**
+ * Converts a value that _may_ be an empty string to a maybe.
+ * If the value is an empty string (""), then Nothing is returned.
```

Comment:
This seems to apply to anything with a length. So it would apply to arrays, too. I wonder if this is an acceptable definition of "empty" for javascript...

We should rewrite the documentation to not be "string-centric"